### ISSUE TITLE
Wrong Account Image with MS

### ISSUE THREAD

**Issue by Teg79** [Erudika/scoold#4]
After logging in with my MS account the account image is in the form:
https://apis.live.net/v5.0/4*****-****-****-****-**********b/picture?type=large

The live.net server responds with:
{
   "error": {
      "code": "request_url_invalid", 
      "message": "The URL contains the path '4afd7ee6-ae59-4719-ab49-3410f48c0c5b', which isn't supported."
   }
}

**Comment by albogdano** [comment]
I'll have a look, the avatar URLs might have changed.

**Comment by albogdano** [comment]
This seems to be related to this: https://social.msdn.microsoft.com/Forums/en-US/843fe8c7-ff7c-4e75-8b23-b956e193177c/windows-live-user-unique-id-using-windows-live-sdk?forum=messengerconnect

My user id is short with a lenght of 16 characters, yours is a UUID, it seems. Try editing `signin.js` file and replace the code for the "Sign in with Microsoft" button with this:

```
	onClick('ms-login-btn', function () {
		window.location = "https://login.microsoftonline.com/consumers/oauth2/v2.0/authorize?" +
                "response_type=code&client_id=" + MICROSOFT_APP_ID +
                "&scope=openid&state=" + APPID +
                "&redirect_uri=" + ENDPOINT + "/microsoft_auth";
		return false;
	});
```

**Comment by albogdano** [comment]
This is fixed in Scoold `1.24.9`, Para `1.24.4`. Profile picture is fetched through the Microsoft Graph API using `https://graph.microsoft.com/beta/me/photo/$value` and stored as Base64 string inside the `picture` fileld of the user object.
